### DISCUSSION TITLE
Does BlueRetro support modern Arcade Sticks?

### DISCUSSION THREAD

**Discussion by HaraldQuake** [darthcloud/BlueRetro#1023] [Q&A]
I got the BlueRetro Wireless Controller Adapter (for PS1/PS2) and want to play some fighting games on PS2 (Capcom vs. SNK2). 

I found this arcade stick on amazon "Arcade Stick, P12 All-Button-Arcade-Controller with Turbo and customizable RGB, Arcade Fight Stick Gaming-Keyboard for Switch/PC/PS4/PS3, supports Hot-Swap and SOCD (P12EU)" by JZW-Shop but it doesn't support PS2. It supports PS3 though. 

Currently I am using my PS3 controller and the BlueRetro Wireless Controller Adapter (for PS1/PS2) to play on my PS2.

Does BlueRetro support modern Arcade Sticks?

**Comment by darthcloud** [comment]
The 8bitdo ARcade Stick now work in latest beta:
https://github.com/darthcloud/BlueRetro/actions/runs/13083210355
